recipes/zziplib/all/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
```

`CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` requires CMake 3.4 at least
```suggestion
cmake_minimum_required(VERSION 3.4)
```